eng/scripts/Create-Package-WorkItem.ps1

```diff
@@ -15,8 +15,7 @@ param (
   [string]$RelatedWorkItemId,
```

Comment:
@praveenkuttappan are we calling this or other script from any other places in the release planner or automation that need to be adjusted?

Reply:
Release planner uses direct Azure DevOps connection to create work item. But Azure DevOps PAT is used by APIView to trigger pipeline runs.

Reply:
Is APIView using this PAT or a different one to trigger pipelines? Also I assume you are planning to move to MI for that work as well.

Reply:
My bad. APIView was already modified to use managed identity to download artifact as well as to run pipeline. https://github.com/Azure/azure-sdk-tools/blob/main/src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs#L91